networks/remote/ansible/setup-journald.yml

```diff
@@ -1,5 +1,7 @@
 ---
 
+#DD_API_KEY
```

Comment:
Does the user have to set this? Can we document how to do so somewhere?

Reply:
Yes, user has to set this. Most of the documentation is missing right now because code-writing was much faster (this code had to be run on our seed nodes already, consider it production code).
I have an issue open to create an overarching documentation for all ansible and terraform scripts which I'm planning to do next week. I ask for some patience until then.
(All ansible scripts that require input parameters have that little comment on top so I don't forget.)

Reply:
BTW, if you want to run it, you add the API key like this:
```ansible-playbook -b -u centos -i inventory/ec2.py -l tag_Environment_whatever -e DD_API_KEY="..." setup-journald.yml```
This will run the code against machines in AWS that are tagged as "whatever", use the `centos` user by default and the user has the ability to sudo to root.